### ISSUE TITLE
no-unused-declaration removes entire import when destructuring using 'as'

### ISSUE THREAD

**Issue by dwilt** [cartant/tslint-etc#21]
Thanks so much for making this command by the way. Super helpful. Found a scenario though where the command will remove an entire `import` line if there is a destructured (unused) variable that is being renamed with ` as `. Here was my scenario:

![image](https://user-images.githubusercontent.com/1065539/58145834-84234f80-7c08-11e9-93f1-fe9e9ed4a701.png)

I was using the `AirImage` import but not `AirImageProps`. After running `tslint` with the following configuration:

`tslint-imports.json`
```json
{
  "extends": ["tslint-etc"],
  "rules": {
    "no-unused-declaration": true
  }
}
```
via this command:

```shell
tslint -c lint/tslint-imports.json --fix --project . 'src/**/*.{ts,tsx}'
```

It removed the entire line _instead of_ making it:

```ts
import AirImage from '../../AirImage';
```

Version: 1.5.2

**Comment by cartant** [comment]
Thanks for the bug report. It contains everything I need.

**Comment by cartant** [comment]
This should be fixed in 1.5.3.
